### ISSUE TITLE
Plural fallback with namespace fails

### ISSUE THREAD

**Issue by ghost** [i18next/i18next-intervalPlural-postProcessor#202]
Currently using Locize and i18next. It seems like there is a problem with using namespaces that are not under the common namespace. The fallback to key2_plural never occurs as described in the documentation.

[FAIL]

			// if a interval is not specified i18next fallbacks to classic plural
			const x1 = t('foo:key2_interval', { postProcess: 'interval', count: 1 }); // -> one item
			const x2 = t('foo:key2_interval', { postProcess: 'interval', count: 4 }); // -> a few items
			const x3 = t('foo:key2_interval', { postProcess: 'interval', count: 100 }); // -> 100 items

[FAIL]

			// if a interval is not specified i18next fallbacks to classic plural
			const x1 = t('key2_interval', { ns:'foo', postProcess: 'interval', count: 1 }); // -> one item
			const x2 = t('key2_interval', { ns:'foo', postProcess: 'interval', count: 4 }); // -> a few items
			const x3 = t('key2_interval', { ns:'foo', postProcess: 'interval', count: 100 }); // -> 100 items

[PASS]

			// if a interval is not specified i18next fallbacks to classic plural
			const x1 = t('key2_interval', { postProcess: 'interval', count: 1 }); // -> one item
			const x2 = t('key2_interval', { postProcess: 'interval', count: 4 }); // -> a few items
			const x3 = t('key2_interval', { postProcess: 'interval', count: 100 }); // -> 100 items

**Comment by jamuhl** [comment]
i think i know why it does not work...will check this tomorrow.

**Comment by jamuhl** [comment]
updating to:

i18next@10.0.3
i18next-intervalplural-postprocessor@0.2.4

should fix the issue

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next-intervalPlural-postProcessor/blob/master/test/interval.spec.js#L45-L51 should cover that case now

**Comment by ghost** [comment]
Will this be problematic to any users? It seems like the latest major version reverted from 1 to 0?

```
-i18next-intervalplural-postprocessor@^1.0.0:
-  version "1.0.0"
-  resolved "https://registry.yarnpkg.com/i18next-intervalplural-postprocessor/-/i18next-intervalplural-postprocessor-1.0.0.tgz#f2112b35edaf35cc4f53d25e03411edf0b70782e"
```

That said, the recommended versions seem to be working! Thanks @jamuhl

**Comment by jamuhl** [comment]
let me check...wasn't the plan...but seems some version get mixed...(was on my not usual pc) doing so

**Comment by jamuhl** [comment]
thank you for reporting just published i18next-intervalplural-postprocessor@1.0.1 so versions is patch from 1.0.0
